right-for-life_front/src/components/ScrollToTop/ScrollToTop.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export function ScrollToTop() {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
```

Comment:
replace with 
window.scrollTo({top: 0, behavior: 'smooth'});

Reply:
Let it be as it is, i think, swich between pages should not be smooth

Reply:
ok